main_with_gui.py

```diff
@@ -154,6 +154,16 @@ def prompt_gpu_location(self, window: QMainWindow):
 
 	def generate_files(self, window: QMainWindow, gpu_loc: GPU):
 		try:
+			check_dep = sp.getstatusoutput([sys.executable, 'check_dependencies.sh'])
+			check_dep = check_dep[0]
+			if check_dep == 1:
+				buttonReply = QMessageBox.question(self, 'Install dependencies',
+												   "There aren't all the dependencies needed; do you want to install them?",
```

Comment:
Make the fact that it won't work without the dependencies clearer in the GUI as well
```suggestion
												   "You need to install some packages in order for the peracotta to work. Do you want to install them?",
```